internal/designate/provider/provider.go

```diff
@@ -160,6 +172,15 @@ func (p designateProvider) Records(ctx context.Context) ([]*endpoint.Endpoint, e
 	return result, nil
 }
 
+func (p designateProvider) supportedRecordType(recordType string) bool {
+	switch recordType {
+	case endpoint.RecordTypeA, endpoint.RecordTypeTXT, endpoint.RecordTypeCNAME, endpoint.RecordTypeNS, endpoint.RecordTypeMX:
```

Comment:
Unfortunately this will need further adjustment as I just noticed we don't even support all record types supported by external-dns (`A, AAAA, CNAME, NS, SRV, TXT`). Not your fault and not part of the change, but since I just looking a little closer at the code, this really needs fixing.